plugins/generator-1.19.2/forge-1.19.2/procedures/deal_damage.java.ftl

```diff
@@ -1,5 +1,5 @@
-<#if field$damagesource?has_content>
-${input$entity}.hurt(${generator.map(field$damagesource, "damagesources")}, ${opt.toFloat(input$amount)});
+<#if input$damagesource?has_content>

```

Comment:
You can't just replace `field$` with `input$`

Reply:
Yes because in `deal_damage.json`, "damagesource" is now an input, not a field